kolibri/plugins/learn/options.py

```diff
@@ -0,0 +1,9 @@
+option_spec = {
+    "LEARN": {
+        "DISABLE_CUSTOM_CHANNEL_NAV": {
+            "type": "boolean",
+            "default": False,
+            "envvars": ("DISABLE_CUSTOM_CHANNEL_NAV",),
```

Comment:
We've also tended to prefix these with `KOLIBRI_` - there's an open issue to just generate these names automatically to avoid this hullabaloo: https://github.com/learningequality/kolibri/issues/5754

If we can just prefix `KOLIBRI_` here for now, would be good.